<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>XIVN1987</author>
 <class>RTTView</class>
 <widget class="QWidget" name="RTTView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>XIVN1987 SEGGER-RTT Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Image/serial.ico</normaloff>Image/serial.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="vLayout">
   <item>
    <widget class="QTextEdit" name="txtMain">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gLayout1">
     <item row="0" column="6">
      <widget class="QPushButton" name="btnDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="btnOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>打开连接</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lblCore">
       <property name="text">
        <string>CPU Core：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="linRTT">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0x20000000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="hSpacer11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="cmbCore">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Cortex-M0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cortex-M0+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cortex-M3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cortex-M4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cortex-M7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cortex-M23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cortex-M33</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cortex-M55</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QCheckBox" name="chkWavShow">
       <property name="text">
        <string>波形显示</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="btnClear">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>清除显示</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QCheckBox" name="chkHEXShow">
       <property name="text">
        <string>HEX 显示</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>JLink_x64.dll路径：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblRTT">
       <property name="text">
        <string>_SEGGER_ 搜索地址：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <spacer name="hSpacer12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QLineEdit" name="linDLL"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gLayout2">
     <item row="0" column="1" rowspan="2">
      <widget class="QPushButton" name="btnSend">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QTextEdit" name="txtSend">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="chkHEXSend">
       <property name="text">
        <string>HEX 发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
